### ISSUE TITLE
failure running TestServer_UDFStreamAgents on OSX

### ISSUE THREAD

**Issue by jonseymour** [influxdata/kapacitor#448]
I received this when running the Kapacitor tests on OSX. Presumably this is because my local python installation is missing the protobuf dependency. 

It would be helpful if the required dependencies could be documented in the comments for the test method.

Probably ideal, would be to run the test in a dockerised container to void the need to install anything other than docker on the client.

```
--- FAIL: TestServer_UDFStreamAgents (32.37s)
	server_test.go:917: open service *udf.Service: failed to load process info for "movingAvg": write error: write |1: broken pipe
2016/04/10 08:40:02 Server listening on /var/folders/q2/_r9_5f1112n7y3704hg3cl2m0000gn/T/kapacitor_server_test911469067/echo.go.sock
2016/04/10 08:40:03 Starting agent for connection 0
2016/04/10 08:40:03 Agent for connection 0 finished
2016/04/10 08:40:03 Starting agent for connection 1
2016/04/10 08:40:03 Agent for connection 1 finished
Traceback (most recent call last):
  File "/Users/jonseymour/ninja/go/src/github.com/influxdata/kapacitor/udf/agent/examples/echo/echo.py", line 3, in <module>
    from agent import Agent, Handler, Server
  File "/Users/jonseymour/ninja/go/src/github.com/influxdata/kapacitor/udf/agent/py/agent.py", line 7, in <module>
    import udf_pb2
  File "/Users/jonseymour/ninja/go/src/github.com/influxdata/kapacitor/udf/agent/py/udf_pb2.py", line 6, in <module>
    from google.protobuf.internal import enum_type_wrapper
```

**Comment by jonseymour** [comment]
I cribbed the dependencies from the Dockerfile:

```
# Install protobuf3
ENV PROTO_VERSION 3.0.0-beta-2
# Download and compile protoc
RUN wget https://github.com/google/protobuf/archive/v${PROTO_VERSION}.tar.gz && \
    tar xf v${PROTO_VERSION}.tar.gz && \
    rm -f v${PROTO_VERSION}.tar.gz && \
    cd protobuf-${PROTO_VERSION} && \
    ./autogen.sh && \
    ./configure --prefix=/usr && \
    make -j $(nproc) && \
    make check && \
    make install

# Install Python Protobuf3
RUN cd protobuf-${PROTO_VERSION}/python && \
    python setup.py install;


```

**Comment by jonseymour** [comment]
Closed via #449
